SECURITY.md

```diff
@@ -19,18 +19,6 @@ The Node.js project engages in an official bug bounty program for security
 researchers and responsible public disclosures.  The program is managed through
 the HackerOne platform. See <https://hackerone.com/nodejs> for further details.
 
-## Reporting a bug in a third party module
-
-Security bugs in third party modules should be reported to their respective
```

Comment:
The first part of this sentence is technically not obsolete.

Reply:
I've restored the relevant part of the text.